core/src/test/scala/org/apache/spark/sql/ViewTestSuite.scala

```diff
@@ -4,23 +4,29 @@ class ViewTestSuite extends BaseTiSparkTest {
   private val table = "test_view"
 
   test("Test View") {
-    tidbStmt.execute(s"drop table if exists $table")
+    tidbStmt.execute(s"create table $table(qty INT, price INT);")
+    tidbStmt.execute(s"INSERT INTO $table VALUES(3, 50);")
+
     try {
-      tidbStmt.execute("drop view if exists v")
```

Comment:
what if there's some dirty data (e.g. a view named `v`) in database?

Reply:
afterAll will drop it.